### PR TITLE
Add scene details modal

### PR DESCRIPTION
**Pull request by Matt95**
Add modal to view scene details including ID and allow changing name without editing JSON.

Also moves delete button one layer down in interface.

### CODE REVIEW

**Comment by Matt95** [comment]
Also fixes some bug's:
- Scenes that are deleted are de-selected (previously kept old scene)
- Scene is now selected upon creation (previously kept old scene)

File: src/js/components/scene-selector.jsx [hunk 1]
```diff
@@ -99,6 +104,35 @@ var SceneSelector = React.createClass({
...
+
+  _closeDetailsModal: function () {
+    this.setState({showDetailsModal: false})
+  },
+
+  _completeDetailsModal: function() {
+    var newName = this.sceneNameEditInput.value;
+    var self = this;
+    //potentialy risky (saving over scene)
+    SceneActions.getFullScene(this.state.sceneListFilter.value._id, function(scene) {
```
Comment (aaronpcz) [inline src/js/components/scene-selector.jsx]: I think this is the wrong call to make, the Full Scene call goes to the asset store to get the database mos appended to it.  

I think using the HubClient - loadSceneWithCb method is a better choice, as it requests the Scene JSON that we store in the scene collection.  The asset store call pulls this scene then adds properties to each mo individually if a record is in either the video, audio or image collections.  We don't actually store this full scene representation for now.

File: src/js/components/scene-selector.jsx [hunk 2]
```diff
@@ -66,22 +69,24 @@ var SceneSelector = React.createClass({
 
 //scene actions
   _createSceneHandler: function(sceneName) {
-    HubSendActions.tryCreateScene(sceneName) //dosn't check for duplicates!
+    var self = this;
+    HubSendActions.tryCreateScene(sceneName, function(newScene) {
+      self.setState({sceneListFilter: {value: newScene, label: newScene.name}})
+    }) //dosn't check for duplicate names!
```
Comment (aaronpcz) [inline src/js/components/scene-selector.jsx]: Personally I think scene names are fine not being unique, especially with our final goal of a SaaS approach with solution and projects.  We will always have a scene id and other indexing opportunities.

File: src/js/components/scene-selector.jsx (hunk 1 above)
Comment (Matt95) [reply src/js/components/scene-selector.jsx]: Make's sense, changed to use the HubClient for load and save during name change.

**Pull request merged**